src/internal/nativeModule.ts

```diff
@@ -7,9 +7,15 @@
  * @format
  */
 
-import {NativeModules} from 'react-native';
-import {NetInfoNativeModule} from './privateTypes';
-
-const RNCNetInfo: NetInfoNativeModule = NativeModules.RNCNetInfo;
-
-export default RNCNetInfo;
+ import {NativeModules} from 'react-native';
+ import {NetInfoNativeModule} from './privateTypes';
+ 
+ // @ts-ignore
```

Comment:
I ts-ignore things sometimes as well, just curious why it is needed here exactly?
If it cannot be avoided, I'd like a comment above containing the exact reason why it is ignored, so that future maintenance might see it and easily understand why it is ignored and under what conditions the ts-ignore may be removed (so those conditions could be casually re-checked over time, and ideally it is removed)

Reply:
that comment is great (it not only taught me something but also had the link for some future person to evaluate it for removal) thanks!

Reply:
I guess it's always better to use `@ts-expect-error` and leave a quick explanation comment with the main reasons why we ignore or expect a TSC error here.

I believe eslint rules also can have validation for having some comment near the `@ts-expect-error` statement